bot.js

```diff
@@ -1,13 +1,15 @@
 // vim: tw=120 softtabstop=4 shiftwidth=4
 
 const split2 = require('split2');
-const { char2num } = require("./utils/char2num");
-const { gtpchar2num } = require("./utils/gtpchar2num");
+
+const { decodeMoves }    = require("./utils/decodeMoves");
+const { gtpchar2num }    = require("./utils/gtpchar2num");
+const { move2gtpvertex } = require("./utils/move2gtpvertex");
 
 const child_process = require('child_process');
-const console = require('./console').console;
-const config = require('./config');
-const { Pv } = require('./pv');
+const console       = require('./console').console;
```

Comment:
Is the extra whitespace on these 3 lines intentional?

Reply:
yes, i wanted to align the equal with the line just above to make it more readable
should i revert it to how it was?